### ISSUE TITLE
Support of default export?

### ISSUE THREAD

**Issue by IRT-fbachmann** [adrai/node-cqrs-domain#101]
Could you add support of default exports in the structureLoader? Sth. like this:
```
if (required.default) {
    required = required.default;
}
```

**Comment by adrai** [comment]
Feel free to submit a PR ;-)
PS. but there is no official support in node right now, right?

**Comment by IRT-fbachmann** [comment]
No, there is no official node support. However, in my structure I liked to have some supporting information associated to the commands/events. I'd like to have that in the same file, which is easier using default export plus extra exports...
PR's coming :)

**Comment by IRT-fbachmann** [comment]
#104
